src/impls.rs

```diff
@@ -43,15 +43,14 @@ impl<T: Config> Pallet<T> {
 			Some(vote) => {
 				// Ensure vote has not already been executed, and vote must be tallied
 				ensure!(!vote.executed, Error::<T>::VoteAlreadyExecuted);
-				ensure!(vote.tally_date > <TimestampOf<T>>::default(), Error::<T>::VoteNotTallied);
+				ensure!(vote.tally_date > 0, Error::<T>::VoteNotTallied);
 				// Ensure vote must be final vote and that time has to be pass after the vote is tallied
 				ensure!(
 					<VoteRounds<T>>::get(vote.identifier).len() == vote.vote_round as usize,
 					Error::VoteNotFinal
 				);
 				ensure!(
-					T::Time::now().saturating_sub(vote.tally_date) >=
-						T::VoteTallyDisputePeriod::get(),
+					Self::now().saturating_sub(vote.tally_date) >= 1 * DAYS,
```

Comment:
I think this should be `Self::now().saturating_sub(vote.tally_date) > 1 * DAYS`, not greater or equal to. Per [original gov](https://github.com/tellor-io/governance/blob/main/contracts/Governance.sol#L189) & [parachain gov](https://github.com/tellor-io/tellor-parachain-contracts/blob/main/src/ParachainGovernance.sol#L351)

Reply:
This looks good to me as-is (we want tally_date to be >0 and time since tally_date to be >= 1 day)